R/additional_indicators.R

```diff
@@ -0,0 +1,129 @@
+#' Add set of extra indicators to pip output
+#'
+#' @param dt data.frame: country level out from PIP
+#'
+#' @return data.frame
+#' @noRd
+additional_indicators <- function(dt) {
```

Comment:
function names should start with a verb. We try to stick to the tidyverse styleguide as much as possible: https://style.tidyverse.org/functions.html

Reply:
Done